templates/index.html

```diff
@@ -16,6 +16,6 @@ <h1 class="title is-2">{{ section.title }}</h1>
     <div class="content">
       {{ section.content | safe }}
     </div>
-  </div>
+</div>
```

Comment:
Wrong identation here

Reply:
Should be fixed, let me know if it's not. I cleaned Bulma steps - too large for very little gain, but I think utterances and donate button are useful.